### ISSUE TITLE
Can I send a message that will delete on both side after one hour?

### ISSUE THREAD

**Issue by bronze1man** [simplex-chat/simplex-chat#1269]
It looks like it only can delete on my side after one day. I can not config the message to deleted on that other side.

**Comment by epoberezkin** [comment]
> Everyone can only act on their own device. By design no one can touch anything in someone else's device.

Yes, I agree with the principle – users should have control over data that is stored on their device. The other party can "ask" for deletion, but whether the request is granted should be down to the user who has the data.
We might be adding a feature when you will be able to allow your trusted contacts to irreversibly delete messages from your device. But it will never be a default.
Ephemeral/Secret/OTR conversations that are coming will also be irreversibly deleted from both devices – they user will have to join such conversation, so it won't be a default too.

> It seems to me, I don't want to be wrong, that only messages not yet read by the recipient are allowed to be deleted.

no, the client allows deletion for 24 hours. It is soft delete though - the message is marked as deleted, but not removed from the database. We don't have the UI to see the original message yet, but it will be added.
